### ISSUE TITLE
Can't export games saves

### ISSUE THREAD

**Issue by locksoft** [FrodeSolheim/fs-uae#342]
Here's the problem: I've played Frontier (Elite 2) for some weeks, and I wanted to transfer my saves on the Switch (uaeall2).
Problem is that fsuae doesn't really saves on the .adf, but instead creates a .sdf file format in another folder.

I tried to add "writable_floppy_images=1" to the config file but nothing changed.

What to do?

Thanks

**Comment by ztronzo** [comment]
Correct! To my experience, its logical to assume an ADF would not be modified if it is marked as read-only, zipped (ZIP) or gunzipped (ADZ)

What happens is the compressed formats are temporary unpacked to allows full read/write but later those changes are discarded after closing the emulator. So yes keep as ADF and make sure you have read/write permissions to it (read only attribute must be unchecked) and keep "writable_floppy_images=1", cause if you dont, then all modifications would be stored a disk layer file (SDF) found under your "save states" folder, which works as a differential file and cannot be used without the original ADF or other emulators.
in theory you may need to attach a new disk ADF, for example in DF1: and copy the files from the Amiga end, or load Frontier, then load the saves and save each of them to the new disk attached.
